test/parallel/test-repl.js

```diff
@@ -818,7 +818,52 @@ const tcpTests = [
       kArrow,
       '',
       'Uncaught:',
-      /^SyntaxError: .* dynamic import/,
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: const comeOn = await import(\'fhqwhgads\');',
+    ]
+  },
+  {
+    send: 'import { export1, export2 } from "module-name"',
+    expect: [
+      kSource,
+      kArrow,
+      '',
+      'Uncaught:',
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: const {export1,export2} = await import(\'module-name\');',
+    ]
+  },
+  {
+    send: 'import * as name from "module-name";',
+    expect: [
+      kSource,
+      kArrow,
+      '',
+      'Uncaught:',
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: const name = await import(\'module-name\');',
+    ]
+  },
+  {
+    send: 'import "module-name";',
+    expect: [
+      kSource,
+      kArrow,
+      '',
+      'Uncaught:',
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: const moduleName = await import(\'module-name\');',
```

Comment:
```suggestion
alternatively use dynamic import: await import(\'module-name\');',
```

Reply:
If we just wanted this, we need no acron and likes? We could have just extracted the label after `from` and wrap it in an `await import(...)`?

Reply:
it depends, i think. if i static import with explicit bindings, or with `* as`, then those need to be used to set the result of `await import()`.

Reply:
For the error message if all that we need is `await import(<module_name>)` it can be a RE?

Reply:
Due to the complexity of import statements and their grammar, I don't think anything short of a parser will be a reliable approach.

If using a parser isn't desirable, then I'm not sure why we'd even bother with more than a link to MDN's page on dynamic import, and they can figure it out for themselves.

Reply:
If the user wrote `import 'module-name'` in REPL, they meant `await import("module-name")`, not `const moduleName = await import('module-name')` (otherwise they would have written `import * as moduleName from 'module-name')`). Using Acorn is probably still more reliable, and more future proof.